### PR TITLE
Prepare for release 1.6

### PR DESCRIPTION
**Pull request by heshamMassoud**
#### Description

Prepare for release 1.6:

- Bumps the new version in the docs.
- Uncomments the new release notes.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 # commercetools sync
 [![Build Status](https://travis-ci.org/commercetools/commercetools-sync-java.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sync-java)
 [![codecov](https://codecov.io/gh/commercetools/commercetools-sync-java/branch/master/graph/badge.svg)](https://codecov.io/gh/commercetools/commercetools-sync-java)
-[![Benchmarks 1.5.0](https://img.shields.io/badge/Benchmarks-1.5.0-orange.svg)](https://commercetools.github.io/commercetools-sync-java/benchmarks/)
+[![Benchmarks 1.6.0](https://img.shields.io/badge/Benchmarks-1.5.0-orange.svg)](https://commercetools.github.io/commercetools-sync-java/benchmarks/)
```
Comment (ahalberkamp) [inline docs/README.md]:
```suggestion
[![Benchmarks 1.6.0](https://img.shields.io/badge/Benchmarks-1.6.0-orange.svg)](https://commercetools.github.io/commercetools-sync-java/benchmarks/)
```

File: docs/RELEASE_NOTES.md [hunk 2]
```diff
@@ -27,17 +27,23 @@
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 
 -->
-
 <!--
-### 1.6.0 -  Oct 8, 2019
+### 1.7.0 -  Oct 17, 2019
```
Comment (ahalberkamp) [inline docs/RELEASE_NOTES.md]: is the date correct?

Comment (heshamMassoud) [reply docs/RELEASE_NOTES.md]: yes, its commented out for the future release (as an example)

File: docs/README.md (hunk 1 above)
Comment (heshamMassoud) [reply docs/README.md]: thaanks @ahalberkamp 🕺

**Pull request merged**